examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_file_batcher_stage.py

```diff
@@ -73,7 +74,7 @@ def on_data(self, file_objects: fsspec.core.OpenFiles):
             ts_and_files.append(TimestampFileObj(ts, file_object))
 
         # sort the incoming data by date
-        ts_and_files.sort()
+        ts_and_files.sort(key=itemgetter(0))
```

Comment:
What does `itemgetter(0)` do?

Reply:
> What does `itemgetter(0)` do?

Returns the first element in the tuple

> Cant you also do something like:
> ts_and_files.sort(key=lambda x: x.timestamp)

Yes, I forgot that they're named tuples, I'll make the change.